### ISSUE TITLE
Expand the domain of `closed` to be applicable to more than just `Question`?

### ISSUE THREAD

**Issue by trwnh** [w3c/activitystreams#594]
as:closed seems like it'd be useful on more than just Question types. say you had a comments section (represented by a Collection probably) and you wanted to declare it to be `closed`. is this semantically different enough to justify a completely different property? it also seems like it wouldn't work with the extension policy, since it would conflict with the existing definition for `closed` as `as:closed`.

tangentially, it seems like the intended usage of `as:closed` is kind of like marking a question as "solved". so maybe that deserves further thought.

**Comment by trwnh** [comment]
> The Question concept in AS2 seems to have never been fully thought through. See https://github.com/w3c/activitystreams/issues/297#issuecomment-216636116, for example.

fair point, it also seems to indicate that `closed` was never thought through either. we have another issue in https://github.com/w3c/activitystreams/issues/542 regarding the range of the property (and of similar issues with the `url` property as defined in the context document). but `closed` being a boolean or a datetime or a graph node (Object/Link) seems like it's doing a lot of things. particularly the last case is what prompted my comment ("this Question is `closed` by (or `closedBy`?) the following object/link)

if the meaning truly were "no further interaction is allowed", then that would be imo annargument for making it more generic and loosening the domain restriction.

**Comment by steve-bate** [comment]
> it also seems to indicate that `closed` was never thought through either

I agree with all those points.

**Comment by evanp** [comment]
This seems like a backwards-compatible change. It's not an error that requires an Errata, but I think it would require a text change.

So: roughly, if it is useful for other types, that needs some documentation. We can expand the domain to `Object` in a future version of the Vocabulary.

**Comment by evanp** [comment]
You can think of this as the decriminalization of using `closed` on other types, and then at some point we can legalize it.
